adapters/displayio/displayio.go

```diff
@@ -0,0 +1,207 @@
+package displayio
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/macros"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"net/http"
+	"text/template"
+)
+
+type DisplayioAdapter struct {
+	endpoint *template.Template
+}
+
+type reqDioExt struct {
+	UserSession string `json:"userSession,omitempty"`
+	PlacementId string `json:"placementId"`
+	InventoryId string `json:"inventoryId"`
+}
+
+func (adapter *DisplayioAdapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+	headers.Add("Accept", "application/json")
+	headers.Add("x-openrtb-version", "2.5")
+
+	var requestExt map[string]interface{}
+	var dioExt reqDioExt
+
+	if len(request.Imp) == 0 {
+		return nil, []error{&errortypes.BadInput{Message: "No impression in the bid request"}}
+	}
```

Comment:
PBS core framework has this check. Adapter code doesn't need to check for it

Reply:
Ok, removed